src/main/java/com/teamwizardry/librarianlib/features/animator/animations/BasicAnimation.kt

```diff
@@ -33,4 +33,23 @@ class BasicAnimation<T : Any>(target: T, property: AnimatableProperty<T>) : Anim
         val new = lerper.lerp(from, to, progress)
         property.set(target, new)
     }
+
+    // builder methods
```

Comment:
```suggestion
    //region - Builder methods
```
(with the other suggestion to close the region of course)